bin/run.js

```diff
@@ -98,6 +101,12 @@ run.watch = function watch() {
 		ignoreInitial: true
 	} );
 
+	this.watchedFiles = walkSync( process.cwd(), {
```

Comment:
This should either be done in `run`, the files array should be manually maintained, or we should just loop over `require.cache` when we re-run and remove any entries that start with our path...

I _believe_ that the current implementation here would never clear the require cache of a file added after `qunit --watch` was ran.  

Specifically, this usage scenario:

* run `qunit --watch tests/**/*-test.js`
* wait for first run to complete
* require a non-existent file from a test
* create that file
* change the file in some way
* note that even though the tests are reran, the changes to that file are not picked up